start.sh

```diff
@@ -16,21 +17,48 @@ export COMPOSE_PROJECT_NAME=${COMPOSE_PROJECT_NAME:-e}
 export SCALE=${SCALE:-3}
 
 run() {
-    echo "Using discovery node for bootstrapping ${SCALE}-node cluster."
+  if [[ -n "${DOCKER_HOST}" ]]; then
```

Comment:
Thanks, I can upgrade that then. I pulled that conditional from the prior implementation.